### DISCUSSION TITLE
SemanticsHandle issue after Flutter latest update to 3.10.2

### DISCUSSION THREAD

**Discussion by proximilian** [leancodepl/patrol#1331] [Q&A]
Hi Guys,

In one of my tests I use Patrol's native feature. So one of the steps of that particular test includes opening a webview and then pressing the phone's  back button. Nothing special at all. Before the update to Flutter 3.10.2 the test was passing successfully. But after the latest update the test runs without any issue till the very end and despite that it fails. The reason is that an instance of SemanticsHandle is active at the end of the test. Having the native part of the test removed, it is passing successfully. Hence I suspect the native feature. Find below the exception itself. Does someone hit the same problem? Can it be some regression of flutter itself?

> java.lang.Exception: ══╡ EXCEPTION CAUGHT BY FLUTTER TEST FRAMEWORK ╞═════════════════
The following assertion was thrown running a test:
A SemanticsHandle was active at the end of the test.
All SemanticsHandle instances must be disposed by calling
dispose() on the SemanticsHandle.

When the exception was thrown, this was the stack:
#0      WidgetTester._verifySemanticsHandlesWereDisposed (package:flutter_test/src/widget_tester.dart:1051:7)
#1      WidgetTester._endOfTestVerifications (package:flutter_test/src/widget_tester.dart:1042:5)
#2      TestWidgetsFlutterBinding._runTestBody (package:flutter_test/src/binding.dart:998:22)
<asynchronous suspension>

**Comment by bartekpacia** [comment]
Duplicate of #1265.

**Reply by proximilian** [reply]
Thanks for letting me know.
Sorry for not looking at the bug backlog before asking. 😕 My bad.
